mitodl/odl-video-service
# Run Youtube upload task every hour

#### What are the relevant tickets?
Closes #702 

#### What's this PR do?
- Changes the youtube upload task to an hourly schedule.  
- Renames `YT_DAILY_UPLOAD_HOUR` env setting to `YT_UPLOAD_FREQUENCY`
- Renames `YT_DAILY_UPLOAD_LIMIT` to `YT_UPLOAD_LIMIT`

#### How should this be manually tested?
- Go to the admin page for a collection with at least a few videos.
- Change `stream_source` to `----` (None) or `Youtube` if it isn't already.
- For `n` videos in that collection, check the `Is public` checkbox for the `Video`
- Set `YT_UPLOAD_FREQUENCY` to 30 and `YT_UPLOAD_LIMIT` to `n-1` in your `.env` file
- Run `docker-compose up` and wait.  Verify that the `cloudsync.tasks.upload_youtube_videos` task is run within about 30 seconds, and that it (tries to) upload `n-1` videos.  The uploads may fail if other settings aren't available, but it should still try `n-1` times.
